engine/robot.py

```diff
@@ -574,12 +584,13 @@ def execute_return(self, base_loc, base_angle, allowed_docking_pos_error, allowe
             allowed_heading_error: the maximum error in radians a robot can have to target heading while turning in
                 place.
         """
-        docking_dist_to_base = 1.0  # how close the robot should come to base before starting DOCKING
+        self.robot_state.prev_phase = Phase.RETURN
```

Comment:
Same as above

Reply:
I think we want it in the beginning so if obstacle avoidance triggers, phase becomes obstacle avoidance phase, but we want to go back to prev phase

Reply:
yeppp